Functions.Templates/Templates-v2/BlobTrigger-Python/template.json

```diff
@@ -0,0 +1,235 @@
+{
+    "author": "Varad Meru (@vrdmr)",
+    "name": "Blob trigger",
+    "description": "$BlobTrigger_description",
+    "programmingModel": "v2",
+    "language": "python",
+    "jobs": [
+        {
+            "name": "Create New Project with BlobTrigger function",
+            "type": "CreateNewApp",
+            "inputs": [
+                {
+                    "assignTo": "$(APP_FILENAME)",
+                    "paramId": "app-fileName",
+                    "defaultValue": "function_app.py",
+                    "required": true,
+                    "condition" : {

```

Comment:
The condition means the `app-fileName` would only run for VSCode, You may want to add portal to this list as well.